packages/api-electron/preload.js

```diff
@@ -23,6 +23,10 @@ class Window {
       features
     });
   }
+
+  static getCurrentWindowId() {
+    return ipc.sendSync('get-window-id');
```

Comment:
Seems odd that we would have to use the main process to get the current window's id - is there not a way to get the window id from the renderer process? Or is this a limitation because of the sandbox?